nix/man_and_less.md

```diff
@@ -0,0 +1,17 @@
+# `vi` movement commands
```

Comment:
Should we add a line that says how we open a manual and why it's important to open it with `less`? 
like for example `ls man less`, since we were using `ls` in the previous lesson.